application/cmd/requester/cmd/root.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2020 Insolar Network Ltd.
```

Comment:
Why this file inside another directory `cmd` in `requester` package?

Reply:
cobra library needs this solution